draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3607,12 +3479,12 @@ https://video.example.com/k/movie1/4/013.ts
    Examples of uCDN triggering activity in dCDN:</t>
 
    <section title="Preposition" anchor="examples-preposition"><t>
-   Below is an example of a CI/T "preposition" command -- a POST to the
-   collection of all Trigger Status Resources.</t>
+   Below is an example of a "preposition" trigger -- a POST to the
+   collection of all triggers.</t>
 
    <t>
     Note that pattern-based specs like "UriPatternMatch" and "UrisRegexMatch" are not allowed
-   in a pre-position Trigger Specification, where dCDN has to have a clear list of objects to obtain.</t>
+   in a preposition Trigger Specification, where dCDN has to have a clear list of objects to obtain.</t>
```

Comment:
It might also be worth mentioning CCIDs, which also don't provide a clear list of objects to obtain.

Reply:
ACK
also fixed spec naming to align with the table (i.e. "uri-pattern-match" instead of UriPatternMatch), this is the only instance